static/src/js/routes/EarthdataDownloadRedirect/EarthdataDownloadRedirect.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
+
+import EarthdataDownloadRedirectContainer from '../../containers/EarthdataDownloadRedirectContainer/EarthdataDownloadRedirectContainer'
+
+import { getEnvironmentConfig } from '../../../../../sharedUtils/config'
+
+export const EarthdataDownloadRedirect = () => {
+  const { edscHost } = getEnvironmentConfig()
+
+  return (
+    <>
+      <Helmet>
+        <title>Earthdata Download Redirect</title>
+        <meta name="title" content="Earthdata Download Redirect" />
+        <meta name="robots" content="noindex, nofollow" />
```

Comment:
Do we need this robots line? I removed it from EarthdataDownload route or is this used for the future google analytics tickets we have?

Reply:
@trevorlang ?